src/common/table/Table.tsx

```diff
@@ -91,13 +92,15 @@ const Table = <D extends object>({
   theme = 'primary',
   globalFilter,
   initialState,
+  autoResetSortBy = true,
```

Comment:
Could we simplify by removing this prop because it seems that we could derive the value here:
`autoResetSortBy = onSortedColChange ? false : true` ?

Reply:
Yes, you're right, atm it's not serving its purpose, I removed it.